src/modules/getBullsAndCows.js

```
@@ -13,7 +13,38 @@
  * Example: { bulls: 1, cows: 2 }
  */
 function getBullsAndCows(userInput, numberToGuess) {
-  /* Write your code here */
+  const user = String(userInput).split('');
+  const correct = String(numberToGuess).split('');
```

Consider adding an explicit defensive check for input lengths (e.g., `if (user.length !== 4 || correct.length !== 4) { throw new Error('Inputs must be 4 digits'); }`) so the function fails fast and predictably when used outside the main app. Right now the loops assume length 4 and could read undefined indexes for invalid inputs.